api/enterprise-api-ref/status-settings.markdown

```diff
@@ -138,7 +158,21 @@ administrator.
     Threshold in minutes that hosts are unreachable before they are considered a health issue.
 * **sameHostsNumberOfRuns** *(integer)*
     Number of samples used to identify a duplicate identity. Default value is 3.
-
+* **enforce2FA** *(boolean)*
+    Determines if two-factor authentication (2FA) is mandatory for all users.
+    If set to `true`, users must enable 2FA; otherwise, they will be locked within 48 hours after the first login.
```

Comment:
If you want to duplicate everything then:

```suggestion
    If set to `true`, users must enable 2FA; otherwise, they will be locked out within 48 hours after the first login.
```